libs/go/zmscli/dump.go

```diff
@@ -182,7 +182,7 @@ func (cli Zms) dumpRoles(buf *bytes.Buffer, dn string) {
 	buf.WriteString(indentLevel1)
 	buf.WriteString("roles:\n")
 	members := true
-	roles, err := cli.Zms.GetRoles(zms.DomainName(dn), &members)
+	roles, err := cli.Zms.GetRoles(zms.DomainName(dn), &members, "", "") //TODO what should be the values here?
```

Comment:
empty strings is the correct value. the client code will skip and treat them as nils